enterprise/migrations/0002_enterprisecustomerbrandingconfiguration.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
All the changes that add this `on_delete` are harmless -- it's already the default. But a Django 2.0 deprecation notice said that we should add this because it will become a required argument. This was extremely tedious but I got it over with.